source/includes/Order/ordertype.md

```diff
@@ -0,0 +1,78 @@
+# Order type
+
+## Available operations.
+
+1. [List orders](#list-orders)
+2. [Create an order](#create-an-order)
+3. [Update an order](#update-an-order)
+4. [Delete an order](#delete-an-order)
+
+### List orders
+
+* Fetch all non-retired order types that match any specified parameters otherwise fetch all non-retired order types. If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.
+
+    ```console
+    GET /ordertype
+     ```
+
+* #### Get a particular order type
+
+    Retrieve a particular order.
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.
+
+    ```console
+    GET /ordertype/:target_ordertype_uuid
+    ```
+
+### Create an order type
+
+* To create an order type you need to specify below attributes in the request body. If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.
```

Comment:
I would suggest including some text like:

> Order types depend on code within the application to properly handle them, so it would be unusual to create a new order type unless some new code (e.g., a module) has been added to the system to handle the new order type.